CRM/Upgrade/Incremental/sql/5.59.alpha1.mysql.tpl

```diff
@@ -0,0 +1,16 @@
+-- Add missing provinces for Luxembourg
+SELECT @country_id := id from civicrm_country where name = 'Luxembourg' AND iso_code = 'LU';
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'CA', 'Capellen');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'CL', 'Clervaux');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'EC', 'Echternach');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'ES', 'Esch-sur-Alzette');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'GR', 'Grevenmacher');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'ME', 'Mersch');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'RD', 'Redange-sur-Attert');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'RM', 'Remich');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'VD', 'Vianden');
+INSERT IGNORE INTO `civicrm_state_province` (`country_id`, `abbreviation`, `name`) VALUES (@country_id, 'WI', 'Wiltz');
+UPDATE `civicrm_state_province` SET abbreviation = 'LU' WHERE name = 'Luxembourg';
+UPDATE `civicrm_state_province` SET abbreviation = 'DI' WHERE name = 'Diekirch';
```

Comment:
While unlikely to match another country, might be best to add `AND country_id = @country_id` to these.

Reply:
added

Reply:
I'm not seeing any any errors thrown by jenkins, can you please explain what you meant with 

> jenkins test this please

comment?

Reply:
@jmacarthur98 can you please rebase this PR as there seems to be a conflict in the `.mysql.tpl` file

Reply:
@seamuslee001 done